nova/conf/cinder.py

```diff
@@ -93,6 +93,18 @@
 cinder.conf), the volume create request will fail and the instance will fail
 the build request.
 By default there is no availability zone restriction on volume attach.
+"""),
+    cfg.IntOpt('min_migration_speed_mib_per_second',
```

Comment:
Since this is only an estimate anyways, I thought integers would be fine, but we can also use floats, if you feel that makes more sense.
Implemented with floats now.